runtime/runtime/src/lib.rs

```diff
@@ -1343,6 +1352,36 @@ impl Runtime {
         Ok(())
     }
 
+    #[cfg(feature = "sandbox")]
+    fn apply_state_patches(
+        &self,
+        state_update: &mut TrieUpdate,
+        states_to_patch: Vec<StateRecord>,
+    ) {
+        for record in states_to_patch {
+            match record {
+                StateRecord::Account { account_id, account } => {
+                    set_account(state_update, account_id, &account);
+                }
+                StateRecord::Data { account_id, data_key, value } => {
+                    state_update.set(TrieKey::ContractData { key: data_key, account_id }, value);
```

Comment:
Why do we check existence of account for contract and not for data or access key?

Reply:
Because for contract code we also update account's code hash. This is same as GenesisStateApplier::apply_batch